api/v1alpha1/lvmcluster_types.go

```diff
@@ -169,24 +169,11 @@ type Condition struct {
 type ConditionType string
 
 const (
-	// ConditionAvailable indicates that the resources maintained by the operator,
-	// is functional and available in the cluster.
-	ConditionAvailable ConditionType = "Available"
-
-	// ConditionProgressing indicates that the operator is actively making changes to the resources maintained by the
-	// operator
-	ConditionProgressing ConditionType = "Progressing"
-
-	// ConditionDegraded indicates that the resources maintained by the operator are not functioning completely.
-	// An example of a degraded state would be if not all pods in a deployment were running.
-	// It may still be available, but it is degraded
-	ConditionDegraded ConditionType = "Degraded"
-
-	// ConditionUpgradeable indicates whether the resources maintained by the operator are in a state that is safe to upgrade.
-	// When `False`, the resources maintained by the operator should not be upgraded and the
-	// message field should contain a human readable description of what the administrator should do to
-	// allow the operator to successfully update the resources maintained by the operator.
-	ConditionUpgradeable ConditionType = "Upgradeable"
+	// ResourcesSynced indicates whether the resources maintained by the operator are reconciled without any issues.
+	ResourcesSynced ConditionType = "ResourcesSynced"
```

Comment:
Nit:
```suggestion
	ResourcesSynced ConditionType = "ResourcesAvailable"
```
We dont just check for reconcile without issues, we also check wether they are available. True, that IS part of the reconcile loop, but the current status "ResourcesSynced" is not really clear on what that means for the operator.